capsul/engine/settings.py

```diff
@@ -273,10 +273,8 @@ def export_config_dict(self, environment=None):
                     env_conf = {}
                     for module in modules:
                         mod_conf = {}
-                        for config in session.configs(module, env):
-                            id = "%s-%s" % (config._id, env)
-                            doc = data[session.collection_name(module)][id]
```

Comment:
To get a value from Storage API, the `get()` method must be called : ` doc = data[session.collection_name(module)][id].get()`

Reply:
oh yes !

Reply:
that's the crucial point I missed! ... OK, now I know how to do it properly :-)